packages/core/app/styles/navi-core/components/visualization-config/series-chart.less

```diff
@@ -0,0 +1,28 @@
+
+.metric-select {
+  margin-bottom: 20px;
+
+  &__header {
+    color: @navi-gray;
+    margin: 5px;
+  }
+
+  &__select {
+    .ember-power-select-status-icon {
+      right: 10px;
+    }
+
+    .ember-power-select-option[aria-current='true'] {
```

Comment:
Delete this block (goodbye purple)

Reply:
Do we need this file now, can we just to the standard navi power select styles?

Reply:
I've removed all the power select styles from this file